content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -104,6 +104,10 @@ Right after `kubeadm init` there should not be any pods in these states.
   likely that the Pod Network solution that you installed is somehow broken. You
   might have to grant it more RBAC privileges or use a newer version. Please file
   an issue in the Pod Network providers' issue tracker and get the issue triaged there.
+- If you install old version of docker, lower than 1.12.1, please remove the 'MountFlags=slave' option
+  when booting dockerd with systemd and restart docker. You can see it in /usr/lib/systemd/system/docker.service
+  systemd MountFlags can interfere with volumes mounted by kubernetes, then make Pods in `CrashLoopBackOff` state.
+  The error is found by "/var/run/secrets/kubernetes.io/serviceaccount/ files not found from kubernets.
```

Comment:
there are missing quotes and typo in Kubernetes.
i don't understand this part.

maybe remove this last sentence?

Reply:
I want to show error messages about this issue to user, then the messages maybe make users puzzled.
OK, i will remove it.

Reply:
perhaps rephrase to:
> The error can happen if `var/run/secrets/kubernetes.io/serviceaccount` files are not found by Kubernetes.
?

Reply:
Another option:
The error happens when Kubernetes does not find `var/run/secrets/kubernetes.io/serviceaccount` files.